### DISCUSSION TITLE
Dynamic Value in the Select Field

### DISCUSSION THREAD

**Discussion by mahirchavda** [pocketbase/pocketbase#860] [Q&A] [answered]
I'm creating my first collection in the pocketbase. I want to create a singleselect field called (category) and multiselect field called (tags). The issue i'm facing is that i have selected both field as multiselect but they are allowing only static values. I looked into document but could not way to create a singleselect/multiselect field that allows custom value from user. I was wondering is it possible to achieve this in the current version of pocketbase?  If not is there any future plan to allow dynamic value in the select field?

**Comment by ganigeorgiev** [comment, marked as answer]
No, the `select` field type works only with predefined options list. You can use the `json` field type to allow any custom value.

Alternatively, you can also move your tags in a separate collections and associated them with a `relation` type field, which will allow you to create as many dynamic tags as you want and still have data validations (including cascading behavior).

---

btw. Please note that the multi-select and multi-relation fields currently has one limitation and that's it that you cannot just append/remove a single value from the array and you have to pass the entire final array every time when you want to update it. For example:
```js
// old myMiltiSelectField is ["1", "2"] and we want to add 3 to the list
client.records.update("example", "RECORD_ID", {
    myMiltiSelectField: ["1", "2", "3"],  
})
```
_This limitation will be fixed after v0.8 when adding support for IN operator(s)._
